### PR TITLE
Feature add skip links accessibility

### PR DESCRIPTION
**Pull request by RachelRVasquez**

### CODE REVIEW

File: _includes/markdown/Markup.md [hunk 1]
```diff
@@ -257,6 +257,21 @@ ARIA attributes can be added with JavaScript to help dynamically add context to
 
 You can see how effortless it is to make our tabbed interface accessible to screen readers. All we need to do is add context.
 
+#### Skip to Content Links
+
+Skip to content links, also known as "skip links", allow screen readers and assistive technologies to skip directly to content. Skip links are a WordPress Core standard that 10up aims to implement across all projects.
```
Comment (ericmann) [inline _includes/markdown/Markup.md]: Perhaps link _here_ to the WordPress standard docs here for a cross-reference? I note that the CSS technique is referenced at the bottom, but in context with the first mention will help anyone unfamiliar with the concept.

File: _includes/markdown/Markup.md [hunk 2]
```diff
@@ -257,6 +257,21 @@ ARIA attributes can be added with JavaScript to help dynamically add context to
...
+
+Skip Links are usually placed in the header of a WordPress theme, before the site navigation. While they usually skip to the main content of the site, they can also skip to other navigations as well as the footer. Skip Links make use of the ```screen-reader-text``` class to position the links off the screen, and on ```:focus```, the links become visible for sighted keyboard users. The CSS technique used to show and hide these links, hide them from sighted users, but will still keep them accessible for screen readers. Here's what a skip link looks like:
+
+```html
+<a class="skip-link screen-reader-text" href="#content">
+  <?php _e( 'Skip to content', 'my-domain' ); ?>
```
Comment (ericmann) [inline _includes/markdown/Markup.md]: `esc_html_e( 'Skip to content', 'my-domain' );` ;-)

**Comment by jonbellah** [comment]
Dig the PR, agree with @ericmann's feedback.

**Comment by tlovett1** [comment]
Would love to hear from @timwright12 on this one.

**Comment by timwright12** [comment]
No objections, skip links are great. There isn't much to them, the only thing we need to make sure of is that the are displayed on :focus. I'll pull the branch and check it out.

**Comment by timwright12** [comment]
+1 looks good to me @tlovett1

**Comment by RachelRVasquez** [comment]
Thanks everyone! 😄  Will make the requested adjustments and re-push soon.

File: _includes/markdown/Markup.md (hunk 1 above)
Comment (RachelRVasquez) [reply _includes/markdown/Markup.md]: I've edited it where there's an additional link. I wanted to keep the one at the end as it links straight to the theme handbook and shows the CSS for it wheres the one I added is more of a general definition for those who aren't familiar with them.

**Comment by RachelRVasquez** [comment]
The adjustments have been made, conflicts resolved and ready to merge. 😄

File: _includes/markdown/Markup.md [hunk 3]
```diff
@@ -257,6 +257,21 @@ ARIA attributes can be added with JavaScript to help dynamically add context to
 
 You can see how effortless it is to make our tabbed interface accessible to screen readers. All we need to do is add context.
 
+#### Skip to Content Links
+
+[Skip to content links](https://developer.wordpress.org/themes/functionality/accessibility/#skip-links), also known as "skip links", allow screen readers and assistive technologies to skip directly to content. Skip links are a WordPress Core standard that 10up aims to implement across all projects.
+
+Skip Links are usually placed in the header of a WordPress theme, before the site navigation. While they usually skip to the main content of the site, they can also skip to other navigations as well as the footer. Skip Links make use of the ```screen-reader-text``` class to position the links off the screen, and on ```:focus```, the links become visible for sighted keyboard users. The CSS technique used to show and hide these links, hide them from sighted users, but will still keep them accessible for screen readers. Here's what a skip link looks like:
```
Comment (tlovett1) [inline _includes/markdown/Markup.md:264]: 1) This paragraph doesn't need to be exclusive to WordPress.

2) Why would a "skip to content link" skip to the footer?

3) This sentence is a bit confusing: "The CSS technique used to show and hide these links, hide them from sighted users, but will still keep them accessible for screen readers."

**Comment by dana-ross** [comment]
@timwright12 This PR is pretty old. Could you review & either let us know what work you think still remains or close it out?

**Comment by timwright12** [comment]
@daveross I think the feature is worth having in the best practice, but we can close the old PR and file a new one since the Markup.md file changed quite a bit.

**Pull request closed without merging**